packages/dso-toolkit/components/Voorbeelden/30-Componenten/viewer-layout-grid/_preview.njk

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="nl">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ _target.label }} | {{ _config.project.title }} - Component preview</title>
+    <link rel="stylesheet" href="{{ '/styles/dso.css' | path }}" />
+    <link rel="stylesheet" href="{{ '/@dso-toolkit/core/dso-toolkit/dso-toolkit.css' | path }}" />
+  </head>
+
+  <style>
+    .dso-map-wrapper {
```

Comment:
Waarom staat deze styling hier en niet in de SCSS? Heeft een consument het niet nodig?

Reply:
Na overleg besloten om deze styling te verplaatsen naar de CSS package